lib/cocoapods-core/podfile/target_definition.rb

```diff
@@ -573,6 +581,36 @@ def store_podspec(options = nil)
         end
       end
 
+      #--------------------------------------#
+
+      SCRIPT_PHASE_REQUIRED_KEYS = [:name, :script].freeze
```

Comment:
minor update to use constants for these to make it easier to read and modify in the future.